media-sound/bitwig-studio/bitwig-studio-1.3.16.ebuild

```diff
@@ -0,0 +1,102 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# TODO
+# - What about bundled libs? Shouldn't they be removed?
+# - Install docs in correct location? (currently in /opt/bitwig-studio/resources/doc)
+
+EAPI=6
+inherit eutils fdo-mime unpacker
+
+DESCRIPTION="Multi-platform music-creation system for production, performance and DJing"
+HOMEPAGE="http://bitwig.com"
+SRC_URI="http://downloads.bitwig.com/${P}.deb"
+LICENSE="Bitwig"
+SLOT="1"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+IUSE="+jack libav"
+
+DEPEND=""
+RDEPEND="${DEPEND}
+	app-arch/bzip2
+	dev-libs/expat
+	media-libs/alsa-lib
+	media-libs/fontconfig
+	media-libs/freetype
+	media-libs/libpng:0/16
+	media-libs/mesa
+	sys-libs/zlib
+	x11-libs/cairo[xcb]
+	x11-libs/libX11
+	x11-libs/libXau
+	x11-libs/libXcursor
+	x11-libs/libXdmcp
+	x11-libs/libXext
+	x11-libs/libXfixes
+	x11-libs/libXrender
+	x11-libs/libdrm
+	x11-libs/libxcb
+	x11-libs/libxkbfile
+	x11-libs/pixman
+	x11-libs/xcb-util-wm
+	virtual/opengl
+	virtual/udev
+	jack? ( virtual/jack )
+	libav? ( media-video/libav )
```

Comment:
Close, but no cigar ;)
You only need to depend on `virtual/ffmpeg`, that's it, no USE flag or conditionals necessary.
See the comments in the ffmpeg virtual ebuild https://github.com/gentoo/gentoo/blob/master/virtual/ffmpeg/ffmpeg-9-r2.ebuild